Sources/MapboxMaps/Foundation/MapView+Attribution.swift

```diff
@@ -7,7 +7,10 @@ extension MapView: AttributionDialogManagerDelegate {
     }
 
     func attributionDialogManager(_ attributionDialogManager: AttributionDialogManager, didTriggerActionFor attribution: Attribution) {
-        let url: URL = attribution.isFeedbackURL ? mapboxFeedbackURL() : attribution.url
+        guard let url: URL = attribution.isFeedbackURL ? mapboxFeedbackURL() : attribution.url else {
```

Comment:
Do we need an explicit `URL` type in this line?

Reply:
No need, indeed. It went away after a refactoring related to https://github.com/mapbox/mapbox-maps-ios/pull/1163#discussion_r817311749